src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import duke.commands.*;
```

Comment:
Similar to importing java classes, it was mentioned by Damith in the forum that wild card imports should not be used for importing packages as well. If you want to, you can see the forum thread [here](https://github.com/nus-cs2103-AY2122S1/forum/issues/73).